### PR TITLE
Update state.json

### PR DESCRIPTION
**Pull request by marksagrouch**
Updating construction schedule for SFO and Pocket

### CODE REVIEW

File: state.json [hunk 1]
```diff
@@ -227,7 +227,7 @@
                 "name": "sfo",
                 "random": true,
                 "commands": [
-                    "gslide id=1ZtS6XVAqrj2pQDPINdXnSVB3_AH1v4wQ_NGwtbD8QPw comment=\"SFO Zoom Transition\""
+                    "gslide id=1VQvRzDCwKOuIYdluBnAYLqN_iMKSKjaz7EaFivkIUTs comment=\"SFO Zoom Transition\""
```
Comment (mythmon) [inline state.json]: It looks like this slide deck isn't publically viewable, so Corsica won't be able to show the images. Can you fix that?

Comment (marksagrouch) [reply state.json]: Both decks are now publicly viewable.  I had published to the web, but they were still locked in my drive.  Fixed

**Comment by mythmon** [comment]
By default, Corsica will choose a random slide from the provided deck. Is this what you want? It also has the ability to choose a specific slide from the deck.

File: state.json [hunk 2]
```diff
@@ -236,13 +236,14 @@
                 "commands": [
                     "gslide id=1gDlOjkeMb9xLHzxhb5P5XL7gpIq08w0hdjCdykRIGTY comment=\"Pocket slideshow\"",
                     "https://forecast.io/embed/#lat=37.779&lon=-122.41499&name=Pocket%20-%20SanFrancisco&color=#DC143C&font=Arial zoom=300 comment=\"SF Weather\""
```
Comment (mythmon) [inline state.json]: You need a comma here for this JSON to be valid.

```suggestion
                    "https://forecast.io/embed/#lat=37.779&lon=-122.41499&name=Pocket%20-%20SanFrancisco&color=#DC143C&font=Arial zoom=300 comment=\"SF Weather\"",
```

**Comment by marksagrouch** [comment]
Random slide selection is totally fine for this submission.  Thanks!

**Pull request merged**